src/components/Alert/Alert.component.jsx

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types'
 import Colors from '../../styles/Colors'
 
 const Container = styled.div`
-    background: pink;
+    background: rgb(255, 192, 203, 0.75);
```

Comment:
if you have a `Colors` constant you can move that color there as well

Reply:
Good idea, I going to change them by constants
:)